examples/rust-examples/xtensa-esp32/wifi/src/main.rs

```diff
@@ -22,7 +22,7 @@ fn setup_fn() {
     println!("Setup hello world!");
     unsafe {
         ESP_NOW = Some(get_esp_now());
-        NEXT_SEND_TIME = Some(current_millis() + 5 * 1000);
+        NEXT_SEND_TIME = Some(time::now().duration_since_epoch().to_millis() + 5 * 1000);
```

Comment:
Why have you decided to change this?

Reply:
I've updated `esp-wifi` dependency, the new version [removed that function](https://github.com/esp-rs/esp-hal/blob/v0.21.0/esp-wifi/MIGRATING-0.9.md#removed-esp_wificurrent_millis).